### DISCUSSION TITLE
Plot Resolution

### DISCUSSION THREAD

**Discussion by monafaraji** [MAIF/shapash#185] [Q&A]
Hi, and thanks for the amazing Shapash, 
I am trying to get results from a data of mine and use Shapash for it, how can I increase the resolution of the png figures that are downloaded? Appreciate any thoughts,

Cheers

**Comment by ThibaudReal** [comment]
Hello monafaraji and thank you for your feedbacks, 

I guess you are talking about the png files that are generated in the Shapash Report (using the `generate_report` method) is that correct ?

For now we did not include an option to change the resolution of the generated png plots, but that could be a great idea.
I will add an issue to better parametrize the Shapash Report and include this idea.

Do you have an example you can share of these low-resolution figures ?

**Reply by monafaraji** [reply]
Thanks for this, I noticed that with smaller length and width the plot resolution can be much better :)

**Reply by monafaraji** [reply]
It would have been great if the font size, titles, and dpi was also adjustable, can't wait for the updaete
thank you
